packaging/agent.conf

```diff
@@ -0,0 +1,57 @@
+agent {
+    data_dir = "./.data"
+    log_level = "DEBUG"
+    server_address = "localhost"
+    server_port = "8081"
+    agent_address = "127.0.0.1"
+    agent_port = "9091"
+    socket_path ="./tmp/spire-agent/public/api.sock"
+    #trust_bundle_path = "./conf/agent/dummy_root_ca.crt"
+    insecure_bootstrap = true
+    trust_domain = "example.com"
```

Comment:
I remember we spoke that the trust_domain is always accuknox.com right ?!

Reply:
This is a test config which will be replaced with the actual config by knoxctl